app/src/main/java/caller/alarmist/AlarmClockWatcher.java

```diff
@@ -362,6 +362,11 @@ public void onCreate() {
         sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
         sharedPreferences.registerOnSharedPreferenceChangeListener(preferenceChangeListener);
 
+        always_notify = sharedPreferences.getBoolean(getString(R.string.pref_key_notif_always), false);
+        wait_for_watch_app = sharedPreferences.getBoolean(getString(R.string.pref_key_wait_for_watch_app), false);
```

Comment:
Hah! That was pretty dumb of me. Good catch.

Reply:
I wasn't actually sure if that functionality worked pre-Oreo or not. :)